### ISSUE TITLE
blue screen blue screen after 10 or 5 min with radar open.

### ISSUE THREAD

**Issue by the7thway** [FutureXpo/PUBG-RADAR-8K#4]
when using radar for like 5-10mins, blue screen pop up, but without it, game runs fine...anyone know the problem is ?

**Comment by FutureXpo** [comment]
Idk, now i am banned for using on same pc, so i cant really help you, but if you want, you can just use version 1.0.6, it works fine

**Comment by the7thway** [comment]
im using hidetask to hide java.exe and it works fine on same PC..played for like 3 days, ,,not banned yet.

**Comment by the7thway** [comment]
and BTW~ is the radar eat alot of ram to run it? i found out that while using radar the game is really lag..

**Comment by the7thway** [comment]
and is it possible that when running radar the CMD windows doesnt pops up? just click on the jar to run the radar.

**Comment by gonhakase** [comment]
how to use hidetask to java.exe on same pc?

**Comment by FutureXpo** [comment]
@the7thway Yeah, it possible to do it(just clicking on the jar), if you want to do so, check [this](https://github.com/AiYinZiLeGong/PUBG-Radar) and modify this Sniffer.kt like there

**Comment by the7thway** [comment]
thx,,,,,,but still have no clue how to do it...lol...will replace the sniffer.kt work?

**Comment by FutureXpo** [comment]
no, you need to change some things in this sniffer

**Comment by the7thway** [comment]
...OMG...do you have some free time to help me out? tell me what to do plz..

**Comment by FutureXpo** [comment]
lol, sry) i am not programmer, i cat really help...

**Comment by the7thway** [comment]
ok..but do you know where i should start with? like change what

**Comment by FutureXpo** [comment]
https://github.com/AiYinZiLeGong/PUBG-Radar/blob/22f97e6716d50cedc553a341835f8caae494636d/src/main/kotlin/pubg/radar/sniffer/Sniffer.kt#L53
this function - init

**Comment by the7thway** [comment]
so from line 53 all the way to the end right?
